packages/components/src/components/modal/modal.js

```diff
@@ -105,7 +105,7 @@ class Modal extends mixin(
       ) {
         (
           this.element.querySelector(this.options.selectorPrimaryFocus) ||
-          this.element
+          this.element.querySelector(settings.selectorTabbable)
```

Comment:
Just wondered if we should have an escape hatch for the condition where `this.element.querySelector(settings.selectorTabbable)` yields `null`.

Reply:
while I understand the concern for this, there should always be at least one button in a modal and buttons should always be in the `selectorTabbable`. Do we want to put an explicit `button` check incase something is overwritten in `selectorTabbable` incorrectly?

Reply:
I think the escape hatch we need is for the edge case where the condition of "a button should already be there" is not met. If you think it's a very edge case it's fine, because it'll be just focus getting lost.

Reply:
I do think that it's a very edge case and there should be something that goes wrong, meaning the focus gets lost, because that means that something isn't implemented correctly. If this happens it means that `selectorTabbable` doesn't have every type of element it should or the Modal doesn't have interactive elements that it should. I do feel like it would be appropriate to throw a console warning, if it comes back null, to inform the developer.  Does Carbon have a standardized way of throwing warnings to developers?

Reply:
Makes sense - We use [`warning` library](https://www.npmjs.com/package/warning) for this purpose.